packages/headless-styles/src/components/Flex/shared.ts

```diff
@@ -0,0 +1,77 @@
+import { type CSSProperties } from 'react'
+import type {
+  FlexDirection,
+  FlexGap,
+  FlexGapPreset,
+  FlexItemOptions,
+  FlexOptions,
+} from './types'
+
+const gapMap: Record<FlexGapPreset, string> = {
+  '6': '0.375rem',
+  '8': '0.5rem',
+  '12': '0.75rem',
+  '16': '1rem',
+  '32': '2rem',
+}
+
+function getGapValue(gap: FlexGap) {
+  if (Object.prototype.hasOwnProperty.call(gapMap, gap)) {
+    return gapMap[gap as FlexGapPreset]
+  }
+  return gap
```

Comment:
I think here we should probably (we can also do a Stacked PR into this branch for doing the same thing in Grid) instead return the REM maths, then just kill the type and `gapMap` since we aren't getting any value from the typing itself:

```suggestion
  return `${gap / 16}rem`
```

Reply:
Then, we can change the type to back to `number`.

Reply:
Ok.  I had it this way to allow entry of any custom value while also providing the "presets." Though it looks like the intellisense doesn't show the presets the way it is now  If I change the presets to numbers, then they show up and we'd still be able to accept both... but I don't know if that causes too much confusion.

If we just go with the number, then the only options would be px converted to rems.  If that's enough, then we can do that, but I thought we wanted to have some more flexibility so I went this route.

Reply:
Ok, thanks. I'll make the suggested updates, then stack the corresponding change to Grid